### PR TITLE
increase version

### PR DESCRIPTION
**Pull request by CarloLucibello**

### CODE REVIEW

**Comment by CarloLucibello** [comment]
@JuliaRegistrator register

File: README.md [hunk 1]
```diff
@@ -59,3 +59,4 @@ julia> for sol in PicoSAT.itersolve(cnf)
 ### Contributors
   * Jake Bolewski - [@jakebolewski](http://github.com/jakebolewski)
   * Carlo Lucibello - [@CarloLucibello](https://github.com/CarloLucibello)
+  * Maxime Bouton -[@MaximeBouton](https://github.com/MaximeBouton)
```
Comment (CarloLucibello) [inline README.md]: @JuliaRegistrator register

**Comment by CarloLucibello** [comment]
@jakebolewski could you install the Registrator and TagBot github apps?

https://github.com/JuliaRegistries/Registrator.jl
https://github.com/JuliaRegistries/TagBot

**Comment by mforets** [comment]
Bump?

**Comment by CarloLucibello** [comment]
@jakebolewski  could you upgrade my permissions so that I can manage the registration thing

**Comment by asinghvi17** [comment]
I believe the PR needs to be merged first, and you can comment off of the commit or an issue.

**Comment by aviks** [comment]
If the github registrator bot is not installed on this repo, the web registrator can bu used by committers to tag the package. Go to juliahub.com, and log in with your github account.

**Pull request merged**